build.gradle

```diff
@@ -32,7 +32,12 @@ allprojects {
 
     tasks.withType(KotlinCompile).all {
         kotlinOptions {
+            jvmTarget = JavaVersion.VERSION_1_8
             allWarningsAsErrors = true
+            freeCompilerArgs += [
+                    "-opt-in=kotlin.RequiresOptIn",
+                    "-Xjvm-default=all-compatibility",
```

Comment:
I don't see this lib to use `JvmDefault` - is there any other reason for adding the `jvm-default` option?

Reply:
Nice catch, I copied the by mistake from another project.